app/src/main/java/com/kickstarter/services/KSApolloClient.kt

```diff
@@ -151,6 +152,34 @@ class KSApolloClient(val service: ApolloClient) : ApolloClientType {
         }
     }
 
+    override fun creatorDetails(slug: String): Observable<CreatorDetails> {
+        return Observable.defer {
+            val ps = PublishSubject.create<CreatorDetails>()
+            service.query(ProjectCreatorDetailsQuery.builder()
+                    .slug(slug)
+                    .build())
+                    .enqueue(object : ApolloCall.Callback<ProjectCreatorDetailsQuery.Data>() {
+                        override fun onFailure(exception: ApolloException) {
+                            ps.onError(exception)
+                        }
+
+                        override fun onResponse(response: Response<ProjectCreatorDetailsQuery.Data>) {
+                            if (response.hasErrors()) {
+                                ps.onError(Exception(response.errors().first().message()))
+                            }
+
+                            response.data()?.project()?.creator()?.let {
+                                ps.onNext(CreatorDetails.builder()
+                                        .backingsCount(it.backingsCount())
+                                        .launchedProjectsCount(it.launchedProjects()?.totalCount() ?: 0)
```

Comment:
Makes sense to coalesce to `0` just noting that this _should_ only ever be 1 or more given that a creator shouldn't get into this state without having created a project 😄.